playbooks/common/openshift-cluster/config.yml

```diff
@@ -25,6 +25,25 @@
   - set_fact:
       openshift_docker_selinux_enabled: "{{ lookup('oo_option', 'docker_selinux_enabled') }}"
     when: openshift_docker_selinux_enabled is not defined
+  - name: Detecting Operating System
+    stat:
+      path: /run/ostree-booted
+    register: ostree_booted
+  # Locally setup containerized facts for now
+  - set_fact:
+      l_is_atomic: "{{ ostree_booted.stat.exists }}"
+  - set_fact:
+      l_is_containerized: "{{ (l_is_atomic | bool) or (containerized | default(false) | bool) }}"
+  - name: Ensure various deps are installed
+    package: name={{ item }} state=present
+    with_items:
+    - python-ruamel-yaml
+    - iproute
+    - python-dbus
+    - python-six
+    - PyYAML
+    - yum-utils
+    when: not l_is_atomic | bool
```

Comment:
I worry about doing this in the playbook here, there are other playbooks that can be called (scaleup, upgrade, cert regen, etc) that would miss installing these dependencies...